com.unity.cinemachine/Runtime/Helpers/CinemachineInputAxisController.cs

```diff
@@ -23,6 +23,20 @@ namespace Unity.Cinemachine
     [HelpURL(Documentation.BaseURL + "manual/InputAxisController.html")]
     public class CinemachineInputAxisController : InputAxisControllerBase<DefaultInputAxisReader> 
     {
+        /// <summary>
+        /// Leave this at -1 for single-player games.
+        /// For multi-player games, set this to be the player index, and the actions will
+        /// be read from that player's controls
+        /// </summary>
+        [Tooltip("Leave this at -1 for single-player games.  "
+                 + "For multi-player games, set this to be the player index, and the actions will "
+                 + "be read from that player's controls")]
+        public int PlayerIndex = -1;
```

Comment:
Is there a need for this with legacy input or it can only be used with the Input System?

Reply:
After checking seems like it is needed only with the input system .